docs/prerequisites/PREREQUISITES-LOCALLY.md

```diff
@@ -0,0 +1,57 @@
+# Prerequisites to installing locally
+
+  1. [Create a Cloud SQL Instance](#create-cloud-sql-instance)
+  1. [Install the protoc compiler](#install-the-protoc-compiler)
+  1. [Install mysql](#install-mysql-compiler)
+  1. [Install virtalenv](#install-virtual-env)
+
+### Create Cloud SQL Instance
+  1. Create a new instance in the [SQL page of GCP console](https://console.cloud.google.com/sql).
+    * Select second generation.
+    * Specify Instance ID.
+    * Select MySQL 5.7 version.
+    * Select db-n1-standard-1 machine type.
+    * Select 25GB storage capacity.
+    * Fill in other details, as desired.
+    * Click Create button.
+  1. Configure Users Access Control
+    * Change password for root user.
```

Comment:
side note: When I created a new Cloud SQL instance yesterday, I was able to specify the root user's password in the same page where you enter the cloud sql instance details.